clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ActiveActiveStoreIngestionTask.java

```diff
@@ -475,15 +475,16 @@ protected void processMessageAndMaybeProduceToKafka(
 
       case UPDATE:
         mergeConflictResult = mergeConflictResolver.update(
-            oldValueByteBufferProvider,
+            oldValueByteBufferProvider.get(),
```

Comment:
Wait, actually I realize this after I 🚢 . Why do we need to remove the lazy here? UPDATE might not need to be applied if RMD indicates it is a fully stale UPDATE?

Reply:
Good point.  I removed the lazy because invoking the lazy is the only way to populate the chunk manifest with a schema id.

Relooking at the code, I think the way to avoid this would be to pass along the ChunkedValueManifestcontainer entirely (instead of calling getManifest).  I'll make that tweak.

Reply:
Added it back.  Now everything works.